src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/UserService.kt

```diff
@@ -191,7 +193,9 @@ class UserService(
       user.email = deliusUser.email.toString()
       user.telephoneNumber = deliusUser.telephoneNumber
       user.deliusStaffCode = deliusUser.staffCode
-
+      user.probationRegion = findProbationRegionFromArea(deliusUser.probationArea)?.apply {
```

Comment:
There is probably a much nicer way to do this. It didn't feel right to copy the create logic and default it to "North West" so if it can't be found, don't update it?

Reply:
Yeah, this feels like a bit of a blunt object. Maybe if the probation region can't be found, we don't update it?

Reply:
Also, this will only run if `userHasChanged` returns true, and we don't currently compare the UserEntity and Delius User's probation code, see https://github.com/ministryofjustice/hmpps-approved-premises-api/blob/b829a1d3d219ec43cf4ec9f74d02bc24fe6090bd/src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/UserService.kt#L311